telemetry/definitions/commonDefinitions.json

```diff
@@ -3538,8 +3538,10 @@
                     "type": "codeTransformSessionId",
                     "required": true
                 },
-                { "type": "result", "required": true },
-                { "type": "reason", "required": true }
+                {
+                    "type": "codeTransformRuntimeError",
+                    "required": false
```

Comment:
Not required here because we use this `codeTransform_isDoubleClickedToTriggerInvalidProject` in many spots, in some of which it is not necessary to include any type of error message.

Reply:
Note that `codeTransformRuntimeError` is an existing field we have (already defined as a string above)

Reply:
The `reason` field should be sufficient to tell you why it failed. Is there a reason you specifically need this field? Can the value that would go in to this field be put in to `reason` instead?

Reply:
If IntelliJ uses the `codeTransformRuntimeError` field already then we should use this field. If not, we should move the error into the `reason` field if possible.